### ISSUE TITLE
Proxy BUG API and DIFF

### ISSUE THREAD

**Issue by LearnMiner** [xmrig/xmrig-proxy#355]
No show port like API#1
Bat file
xmrig-Proxy.exe -r 2 -R 1 -b 0.0.0.0:8081 -o pool -u User -p x -k --api-port 8080 --api-access-token 545

xmrig-Proxy: unknown option -- api-port
xmrig-Proxy: unknown option -- api-access-token
[2019-11-20 02:36:58.421] xmrig-Proxy: unsupported non-option argument '8080'

ABOUT xmrig-proxy/5.0.0 MSVC/2017
LIBS libuv/1.31.0 OpenSSL/1.1.1c
MODE nicehash
POOL #1 pool:port algo auto
BIND #1 0.0.0.0:21553
COMMANDS hashrate, connections, verbose, workers

And bug with DIFF  10000 when i mining with proxy is 1000 - 1500 here with 1 worker (my pc) is 10 000
but minergate is 850845
[2019-11-20 17:23:10.331] #000 new job from pool.supportxmr.com:3333 diff 10000 algo cn/r height 1971139

**Comment by LearnMiner** [comment]
@xmrig

**Comment by xmrig** [comment]
API format changed, use sample config.json or API part for setup wizard https://xmrig.com/wizard

About diff, please rephrase your question, right now it unclear, pool define diff, some of pools can use custom diff (format varies) some of them not, some of pools also set different diff for proxy and you can't change it without additional action.
Thank you.

**Comment by LearnMiner** [comment]
> API format changed, use sample config.json or API part for setup wizard https://xmrig.com/wizard
> 
> About diff, please rephrase your question, right now it unclear, pool define diff, some of pools can use custom diff (format varies) some of them not, some of pools also set different diff for proxy and you can't change it without additional action.
> Thank you.

Yes, but... why proxy with 3 connect is in minergate this? 800105
 #000 use pool xmr.pool.minergate.com:45700  136.243.102.154
[2019-11-21 16:38:14.200] #000 new job from xmr.pool.minergate.com:45700 diff 800105 algo cn/r height 1971878

is all time 800105 with 1 connect or 3 always 800105 is a bugg? supportmonero is 10000

**Comment by xmrig** [comment]
It not a bug, minergate detects proxy by user agent and set higher diff, 800105 seems to be OK as start value to around 10 kH/s, if you have less, likely you don't need any proxy.

You can set `"user-agent": "XMRig/5.0.0",` to make pool think it a regular single miner.
